src/Microsoft.OpenApi.Readers/V2/OpenApiDocumentDeserializer.cs

```diff
@@ -243,6 +252,26 @@ private static void FixRequestBodyReferences(OpenApiDocument doc)
                 walker.Walk(doc);
             }
         }
+
+        private static bool IsHostValid(string host)
+        {
+            try
+            {
```

Comment:
Remove the try-catch since we don't expect any errors here. If there's a bug, swallowing the error will make it difficult to spot.